### ISSUE TITLE
Feature request: Quantize Google Gemma models

### ISSUE THREAD

**Issue by younesbelkada** [Vahe1994/AQLM#29]
Hi authors! 

With the recent AQLM integration in transformers, would it makes sense to quantize the Google gemma models in 2-bit 

The list of the models can be found here: https://huggingface.co/collections/google/gemma-release-65d5efbccdbb8c4202ec078b

cc @BlackSamorez

**Comment by Godofnothing** [comment]
Hi, @younesbelkada. We intend to prepare 2bit gemma models and will come back to you once we have results.

**Comment by younesbelkada** [comment]
Thanks a lot @Godofnothing ! Looking forward to it !

**Comment by Godofnothing** [comment]
Hi, @younesbelkada. We have uploaded 2b gemma versions to the huggingface hub:
- [gemma-2b-AQLM-2Bit-1x16-hf](https://huggingface.co/BlackSamorez/gemma-2b-AQLM-2Bit-1x16-hf)
- [gemma-2b-AQLM-2Bit-2x8-hf](https://huggingface.co/BlackSamorez/gemma-2b-AQLM-2Bit-2x8-hf)
For some reason, 7b model gemma experiences significant decline in performance, making it unusable. If we managed to resolve the issue, we will upload 7b models as well.
